at_client/lib/src/response/at_notification.dart

```diff
@@ -24,7 +29,9 @@ class AtNotification {
       'to': to,
       'epochMillis': epochMillis,
       'value': value,
-      'operation': operation
+      'operation': operation,
+      'messageType': messageType,
```

Comment:
What is 'messageType'?

Reply:
The message is to indicate if the notification is of type Key or Text.